src/components/Match/renderMatch.js

```diff
@@ -266,13 +266,15 @@ function renderMatch(m) {
   });
 
   if (m.objectives && m.objectives.length > 0) {
-    m.objectives.map(obj => (
-      obj.player_slot === undefined && (
-        obj.slot > 4
-          ? Object.defineProperty(obj, 'player_slot', { value: obj.slot + 123 })
-          : obj.slot > 0 && Object.defineProperty(obj, 'player_slot', { value: obj.slot })
-      )
-    ));
+    m.objectives.forEach((obj) => {
+      if (obj.slot >= 0 && obj.player_slot === undefined) {
+        if (obj.slot > 4) {
+          Object.defineProperty(obj, 'player_slot', { value: obj.slot + 123 });
```

Comment:
We can put obj.slot > 0 as a check in the function, but I think it's probably better to create a new array rather than circumvent the spirit of the eslint rule by avoiding the use of =.

Object.defineProperty will mutate the input, correct?